opengever/apiclient/client.py

```diff
@@ -72,3 +74,16 @@ def listing(self, name, columns=[], raw=False, **kwargs):
 
     def update_object(self, **data):
         return self.session().patch(self.url, json=data).ok
+
+    @autowrap
+    def create_document(self, title, io, content_type, filename):
```

Comment:
`io` will shadow an eventual import of the Python `io` module. Could we name it differently?

Reply:
I named it `fio` but didn't stage that change in the client file, only in the test.

Reply:
I guess `fio` isn't really better, what's the python convention on how to do that?

Reply:
I don't think there is a Python convention. What about `content` or `data`?

Reply:
`content` or `data` is the thing which comes out of the `io`. 

`ioish`? 😂

Reply:
🤔 Any input from @sebastianmanger or @lvonlanthen ?